src/cli/generator/config.rs

```diff
@@ -25,6 +25,8 @@ pub struct Config<Status = UnResolved> {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub preset: Option<PresetConfig>,
     pub schema: Schema,
+    #[serde(default, skip_serializing_if = "TemplateString::is_empty")]
```

Comment:
unnecessary changes.

Reply:
existing tests don't even work rn.. the config in tests is either outdated or we need to add the `default` tag on `secret`